src/core/qgsdistancearea.h

```diff
@@ -222,6 +222,21 @@ class CORE_EXPORT QgsDistanceArea
      */
     double measureLine( const QgsPoint &p1, const QgsPoint &p2, QgsUnitTypes::DistanceUnit &units ) const;
 
+    /**
+     * calculates distance from one point with distance in meters and azimuth (direction)
+     * based on inverse Vincenty's formula when Geographic
+     * otherwise QgsPoint.project() will be called after QgsUnitTypes::fromUnitToUnitFactor() has been applied to the distance
+     * note: usage:
+     *  QgsDistanceArea myDa;
+      * myDa.setSourceCrs( layer->crs() );
+     * \since QGIS 3.0
+     * @param p1 start point [can be Cartesian or Geographic]
```

Comment:
Can you use \param instead of @param? A week or so ago the doxygen strings were all standardised to use the \ notation rather than @ notation.